src/utils/front/ga.ts

```diff
@@ -0,0 +1,46 @@
+declare global {
+  interface IGaEvent {
+    hitType: 'event'
+    eventCategory: string
+    eventAction: string
+    eventLabel?: string
+  }
+
+  interface IGaException {
+    hitType: 'exception'
+    exDescription: string
+    exFatal: boolean
+  }
+
+  // eslint-disable-next-line @typescript-eslint/interface-name-prefix
+  interface Window {
+    ga: (command: 'send', data: IGaEvent | IGaException) => void
```

Comment:
Should be `ga?`. Ad blockers and privacy extensions cause `ga` to not load.

Reply:
`window.ga` is set by `gatsby-plugin-google-analytics`. I only described its interface